src/banners/HomepageDeprecationBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/css';
+import { Alert, useStyles2 } from '@grafana/ui';
+import React, { useEffect, useState } from 'react';
+
+const HP_BANNER_KEY = 'homepageDeprecationBanner';
```

Comment:
nit: could be helpful to namespace this

```suggestion
const HP_BANNER_KEY = 'metricsDrilldownHomepageDeprecationBanner';
```